automatic/mono/mono.nuspec

```diff
@@ -26,7 +26,7 @@
 - [Comprehensive Technology Coverage][]: Bindings and managed implementations of many popular libraries and protocols.
 
 ## Notes
-- Starting from package version 5.8.0.127 and onwards 64bit is supported, unfortunately the 64bit version does not contain the GTK# runtime. Please either install the 32bit edition (`chocol install mono --x86`), or the [gtksharp](https://chocolatey.org/packages/gtksharp) package if this is needed.
+- Starting from package version 5.8.0.127 and onwards 64bit is supported, unfortunately the 64bit version does not contain the GTK# runtime. Please either install the 32bit edition (`choco install mono --x86`), or the [gtksharp](https://chocolatey.org/packages/gtksharp) package if this is needed.
```

Comment:
Just an FYI, you don't need to update the description in the `.nuspec` files for automatic packages. These will be automatically updated from the Readme.md file the next time the package updates.

Reply:
I "thought" that was the case, but I wasn't sure, so I thought I best do it just to be sure. Thanks for confirming!  Not sure if/when there will be a new release of mono, so this change might never make it out, but still, better to fix it than not.